test/hcal_digi_pipeline_test_config.py

```diff
@@ -0,0 +1,34 @@
+
+from LDMX.Framework import ldmxcfg
+
+# Create a process
+p = ldmxcfg.Process( 'test_hcal_digis' )
+
+# Set the maximum number of events
+p.maxEvents = 10 # should be the same as NUM_TEST_SIM_HITS
```

Comment:
This is not the same as NUM_TEST_SIM_HITS, which means you are only testing the lowest 10 sim hits.

Reply:
I increased the number of test Sim hits to [1000](https://github.com/LDMX-Software/Hcal/blob/c365900960ae04df46f00caf2f94cad402d9f3b6/test/HcalDigiPipelineTest.cxx#L60), also reflected [here](https://github.com/LDMX-Software/Hcal/pull/5/commits/5a7474b035dfff9b716b795a30c07e008987efc3).